src/metamask/sign.ts

```diff
@@ -16,3 +16,17 @@ export const sign =
 
     await clickOnButton(page, "Sign");
   };
+
+export const signTypedData =
+  (page: DappeteerPage, getSingedIn: GetSingedIn) =>
+  async (): Promise<void> => {
+    await page.bringToFront();
+    if (!(await getSingedIn())) {
+      throw new Error("You haven't signed in yet");
+    }
+    await page.reload();
+    await page.waitForTimeout(300);
+    await clickOnLittleDownArrowIfNeeded(page);
+    await page.waitForTimeout(300);
```

Comment:
can we instead wait for sign button to not be disabled?

Pretty sure there is some evalueFunction or something method

Reply:
absolutely, I'm pretty sure this is all unneeded. I'll refine this.